src/Client/Web/Bit.Client.Web.BlazorUI/Components/NumericTextField/BitNumericTextField.razor.cs

```diff
@@ -623,7 +639,7 @@ protected override void Dispose(bool disposing)
         }
 
         /// <inheritdoc />
-        protected override bool TryParseValueFromString(string? value, [MaybeNullWhen(false)] out TValue? result, [NotNullWhen(false)] out string? validationErrorMessage)
+        protected override bool TryParseValueFromString(string? value, [MaybeNullWhen(false)] out TValue result, [NotNullWhen(false)] out string? validationErrorMessage)
```

Comment:
this change should be revoked, the result needs to be nullable or there will be a permanent warning:

`Parameter 'result' must have a non-null value when exiting with 'true'.`